install.sh

```diff
@@ -250,9 +251,9 @@ function extract_install () {
     cd "${CREW_DEST_DIR}"
 
     #extract and install
-    echo "Extracting ${1} (this may take a while)..."
+    echo -e "Extracting ${1} (this may take a while)..."
     tar xpf ../"${2}"
-    echo "Installing ${1} (this may take a while)..."
+    echo -e "Installing ${1} (this may take a while)...${RESET}"
```

Comment:
Missing `${BLUE}` at the start of the string?

Reply:
If the color is white already why do we need the `${RESET}`?

Reply:
Removed